etl/steps/data/garden/artificial_intelligence/2025-11-07/energy_ai_iea.meta.yml

```diff
@@ -0,0 +1,77 @@
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Artificial Intelligence
+        - Energy
+  base_case: &base_case
+    - The Base Case looks at how electricity use in data centres is expected to change if today’s regulations stay the same and the industry develops as currently projected.
+    - In the short term, the main factor shaping this outlook is industry forecasts for server shipments up to 2028, with similar trends assumed to continue beyond that point.
+    - Improvements in energy efficiency are expected to play a key role in keeping electricity use from rising too quickly, even as demand for digital services continues to grow.
+  data_centers_description: &data_centers_description
+    - Data centers house the servers and computing infrastructure that power internet services, cloud computing, and artificial intelligence applications.
+    - This metric includes electricity consumption for all data center operations, encompassing both conventional servers and accelerated servers used for AI and machine learning workloads.
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+tables:
+  energy_ai_iea:
+    variables:
+      total_electricity_consumption__twh__historical:
+        title: Total electricity consumption by data centers
+        unit: terawatt-hours
+        short_unit: TWh
+        description_short: Electricity consumption by data centers, measured in [terawatt-hours](#dod:watt-hours). Data centres run the full spectrum of internet services from shopping to streaming, not solely AI. The exact split between AI and other uses is not available.
+        description_key: *data_centers_description
+        display:
+          numDecimalPlaces: 0
+
+      total_installed_capacity__gw__historical:
+        title: Total installed capacity for data centers
+        unit: gigawatts
+        short_unit: GW
+        description_short: Total installed power capacity of data centers worldwide and by region.
+        description_key: *data_centers_description
+        display:
+          numDecimalPlaces: 1
+
+      total_electricity_consumption__share_of_total_electricity_demand__historical:
+        title: Share of total electricity demand consumed by data centers
+        unit: "%"
+        short_unit: "%"
+        description_short: Data center electricity consumption as a percentage of total global or regional electricity demand.
```

Comment:
I think the "total global or regional" is confusing. The "global" applies only to "World", as far as I understand. Maybe something like:
Data center electricity consumption as a percentage of total electricity demand of the country or region.